acceptance_tests/utilities/event_helper.py

```diff
@@ -201,3 +190,26 @@ def _check_new_uacs_are_as_expected(emitted_uacs: List[Mapping], active: bool, f
         if field_to_test:
             test_helper.assertEqual(uac[field_to_test], expected_value,
                                     f"UAC {uac} field {field_to_test} doesn't equal expected {expected_value}")
+
+
+@retry(wait=wait_fixed(1), stop=stop_after_delay(30))
+def check_if_event_types_are_exact_match(expected_logged_event_types, case_id):
+    logged_events = get_logged_events_for_case_by_id(case_id)
+    actual_logged_event_types = [event['type'] for event in logged_events]
+
+    test_helper.assertCountEqual(expected_logged_event_types, actual_logged_event_types,
+                                 msg=f"Actual logged event types {actual_logged_event_types} "
+                                     f"did not match expected {expected_logged_event_types}")
+
+
+@retry(wait=wait_fixed(1), stop=stop_after_delay(30))
```

Comment:
Why does this have a retry?

Reply:
Accidentaly left over when moving stuff about